resources/js/Pages/Admin/BookingRequests/BookingRequestsList.vue

```diff
@@ -161,12 +170,18 @@ export default {
         });
     },
     calendar(timestamp) {
-        return moment(timestamp).local().format('LLL');
+      return moment(timestamp).local().format('LLL');
```

Comment:
Good, I believe it's not set as a standard through the app but we should be sticking to LLL format when showing dates for simplicity.